src/components/admin/form-components/withSearchBar.jsx

```diff
@@ -1,223 +1,94 @@
 import React from 'react';
-import BaseComponent from 'lib/BaseComponent';
-import { capFirstLetter, slugify, debounce } from 'lib/utilities';
+import PropTypes from 'prop-types';
 import _ from 'lodash';
 
-import { updateFieldValue } from 'components/admin/lib/form-field-updaters';
 import { cleanupFalcorKeys } from 'lib/falcor/falcor-utilities';
+import { debounce } from 'lib/utilities';
+import { getDisplayName } from 'lib/higher-order-helpers';
 
-// material-ui
-import TextField from 'material-ui/TextField';
-import Menu from 'material-ui/Menu';
-import MenuItem from 'material-ui/MenuItem';
-import ContentAdd from 'material-ui/svg-icons/content/add';
-
-const withSearchBar = config => {
-  if (!('formatter' in config) || !_.isFunction(config.formatter)) {
-    throw new Error(
-      'withSearchBar.config.formatter must be supplied as a function.',
-    );
-  }
-  if (!('fields' in config) || !_.isArray(config.fields)) {
-    throw new Error(
-      'withSearchBar.config.fields must be supplied as an array.',
-    );
-  }
-  if (!('displayName' in config) || !_.isString(config.displayName)) {
-    throw new Error(
-      'withSearchBar.config.displayName must be supplied as a string.',
-    );
-  }
-  if (!('category' in config) || !_.isString(config.category)) {
-    throw new Error(
-      'withSearchBar.config.category must be supplied as a string.',
-    );
-  }
-
-  class SearchBar extends BaseComponent {
+export const withSearchableConcept = (
```

Comment:
Either change the file name or the export name here so that they match.

Reply:
@frederikbrinck Don't know why these changes are showing outdated. If they are really outdated, my bad. Otherwise, don't miss these "outdated" changes!

Reply:
Yup, good catch :)